public/js/actions/index.js

```diff
@@ -0,0 +1,25 @@
+import fetch from 'unfetch';
```

Comment:
I'd call this file what the exported function does, just for avoidance of doubt when including it.